ui/js/app.js

```diff
@@ -42,14 +42,26 @@ var App = React.createClass({
   _upgradeDownloadItem: null,
   _version: null,
 
-  // Temporary workaround since electron-dl throws errors when you try to get the filename
+  getUpdateUrl: function() {
+    switch (process.platform) {
+      case 'darwin':
+        return 'https://lbry.io/get/lbry.dmg';
+      case 'linux':
+        return 'https://lbry.io/get/lbry.deb';
+      case 'win32':
+        return 'https://lbry.io/get/lbry.exe'; // should now be msi
```

Comment:
I was going on Job's old comment about how the installer is now an MSI (deleted in this PR). But I guess that was written before we switched to Electron, and we're back to an exe.